sample/DokanNetMirror/DokanNetMirror.csproj

```diff
@@ -14,6 +14,10 @@
     <PackageReference Include="StringInterpolationBridgeStrong" Version="0.9.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>

```

Comment:
This does not looks to be needed.

Reply:
Oh, that's right. I did not notice that!